cmd/internal/shared/applicationserver/config.go

```diff
@@ -18,12 +18,13 @@ import (
 	"time"
 
 	"go.thethings.network/lorawan-stack/pkg/applicationserver"
+	"go.thethings.network/lorawan-stack/pkg/applicationserver/io/mqtt"
 )
 
 // DefaultApplicationServerConfig is the default configuration for the Application Server.
 var DefaultApplicationServerConfig = applicationserver.Config{
 	LinkMode: "all",
-	MQTT: applicationserver.MQTTConfig{
+	MQTT: mqtt.Config{
 		Listen:    ":1883",
 		ListenTLS: ":8883",
```

Comment:
I think we should also set default public addresses in here, like we do with the public addresses in the email and canonical URLs in the webui configurations.